.idea/runConfigurations/Artemis__Server_.xml

```diff
@@ -1,13 +1,12 @@
 <component name="ProjectRunConfigurationManager">
```

Comment:
Can you please revert the changes in this file (and keep them only local) so that other developers still have the bamboo / bitbucket / jira profiles here.